src/Spout/Writer/Common/Entity/Cell.php

```diff
@@ -81,6 +100,25 @@ public function getValue()
     }
 
     /**
+     * @param Style $style
+     */
+    public function setStyle(Style $style)
+    {
+        $this->style = $style;
+    }
+
+    /**
+     * @return Style
+     */
+    public function getStyle()
+    {
+        if (!isset($this->style)) {
```

Comment:
I did this so there can be unstyled cells. Only when a writer / worksheet / manager requests a style it should be created. This will probably change though with the Cell Manager concept...